code/modules/deathmatch/deathmatch_modifier.dm

```diff
@@ -554,3 +568,33 @@
 		return
 
 	SSquirks.AssignQuirks(player, player.client)
+
+/datum/deathmatch_modifier/martial_artistry
+	name = "Random martial arts"
+	description = "Everyone learns a random martial art!"
+	blacklisted_maps = list(/datum/lazy_template/deathmatch/meatower)
+	// krav maga excluded because its too common and too simple, mushpunch excluded because its horrible and not even funny
+	var/static/list/weighted_martial_arts = list(
+		// common
+		/datum/martial_art/cqc = 30,
+		/datum/martial_art/the_sleeping_carp = 30,
+		// uncommon
+		/datum/martial_art/boxing/evil = 20,
+		// LEGENDARY
+		/datum/martial_art/plasma_fist = 5,
+		/datum/martial_art/wrestling = 5, // wrestling is kinda strong ngl
+		/datum/martial_art/psychotic_brawling = 5, // a complete meme. sometimes you just get hardstunned. sometimes you punch someone across the room
+	)
+
+/datum/deathmatch_modifier/martial_artistry/apply(mob/living/carbon/player, datum/deathmatch_lobby/lobby)
+	. = ..()
+
+	var/datum/martial_art/picked_art_path = pick_weight(weighted_martial_arts)
+	var/datum/martial_art/instantiated_art = new picked_art_path()
+
+	if (istype(instantiated_art, /datum/martial_art/boxing))
+		player.mind.adjust_experience(/datum/skill/athletics, SKILL_EXP_LEGENDARY)
```

Comment:
Wouldn't this autogrant people the athletics achievement?

Reply:
it didn't do so locally but also it was a local test

Reply:
there is no achievement for this